funnel/models/event.py

```diff
@@ -2,10 +2,12 @@
 from . import db, BaseMixin
 from .space import ProposalSpace
 from .user import User
-import random
-import uuid
 from sqlalchemy.ext.associationproxy import association_proxy
 from datetime import datetime
+from ..util import get_rows_from_csv, format_twitter
+import logging
+import base64
+import os
```

Comment:
As a convention, imports should be in the order of core Python libraries, extended Python libraries, third party libraries, related project libraries and internal imports. That means `os` should be the very first import, followed by `base64`, `logging`, `datetime`, `sqlalchemy` and then the project imports.